CMake/unix_config.cmake

```diff
@@ -1,6 +1,14 @@
 message(STATUS "Setting Unix configurations")
 
 macro(os_set_flags)
+
+    # Put all the collaterals together, so we can find when trying to run examples/tests
+    # Note: this puts the outputs under <binary>/<build-type>
+    # Note note: this ruins our CI procedures... otherwise it mirrors Windows, simplifies
```

Comment:
Please add a prefix 'We cannot uncomment because this.."

Reply:
A prefix?

Reply:
```
  # Note note: this ruins our CI procedures... otherwise it mirrors Windows, simplifies
```
to
```
  # Note note: We cannot uncomment because it ruins our CI procedures... otherwise it mirrors Windows, simplifies
```

Reply:
Changed